docs/roadmap.md

```diff
@@ -16,8 +16,9 @@ Still in case something goes wrong we are not responsible for it, we will try ou
 
 While its technically possible to remove macOS entirely, its strongly encouraged to dual boot it with Linux because :-
 
-1. It acts as a backup in case something goes wrong.
-2. macOS updates often bring along certain firmware updates, which tend to be useful for Linux as well.
+1. To set up Wi-Fi on Linux, you will need Wi-Fi firmware, which can be obtained only from macOS.
```

Comment:
maybe say "can be only legally obtained"? to provide a little context as to why the files can't be hosted on a website

Reply:
Done